### PR TITLE
Rename spec to 'Selective Disclosure for JWTs'

### PR DESCRIPTION
**Pull request by danielfett**
PR for https://github.com/oauthstuff/draft-selective-disclosure-jwt/issues/125

### CODE REVIEW

**Comment by danielfett** [comment]
Done, and also cleaned up the intro slightly (repeating 'document' is superfluous there).

File: main.md [hunk 1]
```diff
@@ -42,10 +42,10 @@ documents that support selective disclosure of JWT claim values.
 
 # Introduction {#Introduction}
 
-The JSON-based claims in a signed JSON Web Token (JWT) [@!RFC7519] document are
+The JSON-based representation of claims in a signed JSON Web Token (JWT) [@!RFC7519] is
 secured against modification using JSON Web Signature (JWS) [@!RFC7515] digital
-signatures. A consumer of a signed JWT document that has checked the document's
-signature can safely assume that the contents of the document have not been
+signatures. A consumer of a signed JWT that has checked the token's
```
Comment (Sakurann) [inline main.md:47]: `token`..?

Comment (danielfett) [reply main.md:47]: ...but the T in JWT already stands for 'token'...

Comment (Sakurann) [reply main.md:47]: yes, but I don't think many people call JWT a token...

Comment (Sakurann) [inline main.md:47]:
```suggestion
signatures. A consumer of a signed JWT that has checked the
```

**Pull request merged**